tests/lax_numpy_test.py

```diff
@@ -1379,10 +1379,41 @@ def np_fun(lhs, rhs):
     tol = max(jtu.tolerance(lhs_dtype, tol_spec),
               jtu.tolerance(rhs_dtype, tol_spec))
     # TODO(phawkins): there are float32/float64 disagreements for some inputs.
-    self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=False,
-                            tol=tol)
-    self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=False, atol=tol,
-                          rtol=tol)
+    self._CheckAgainstNumpy(np_fun, jnp_fun, args_maker, check_dtypes=False, tol=tol)
+    self._CompileAndCheck(jnp_fun, args_maker, check_dtypes=False, atol=tol, rtol=tol)
+
+
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name": "_{}_deg={}_rcond={}_full={}_w={}_cov={}".format(
+          jtu.format_shape_dtype_string(shape, dtype),
+          deg,
+          rcond,
+          full,
+          w,
+          cov),
+      "shape": shape, "dtype": dtype, "deg": deg,
+      "rcond": rcond, "full": full, "w":w, "cov":cov}
+      for dtype in [dt for dt in float_dtypes if dt not in [jnp.float16, jnp.bfloat16]]
+      for shape in [shape for shape in one_dim_array_shapes if shape != (1,)]
+      for deg in jnp.array([1, 2, 3])
```

Comment:
This should be `for deg in [1, 2, 3]`. Trying to use `jnp.array` in the initial parsing of the files leads to errors in some backends because XLA has not yet been initialized.

Reply:
made the change